src/NexusMods.Icons/UnifiedIcon.cs

```diff
@@ -81,6 +81,10 @@ protected override void OnPropertyChanged(AvaloniaPropertyChangedEventArgs chang
 
             MaxHeight = value;
             MaxWidth = value;
+        } else if (change.Property == ForegroundProperty && Content is PathIcon)
+        {
+            // Note(Al12rs): workaround to update PathIcon foreground colors
+            ((PathIcon)Content).Foreground = (IBrush)change.NewValue!;
```

Comment:
This shouldn't be necessary.

Reply:
Yeah, that is what I thought as well, but without this the icon only takes the initial foreground and doesn't respond to updates. So this is the workaround